src/extra/AutoUi/Collection/index.tsx

```diff
@@ -320,6 +337,111 @@ export const AutoUICollection = <T extends AutoUIBaseResource<T>>({
 									filters={filters}
 									autouiContext={autouiContext}
 									changeFilters={setFilters}
+									onSearch={(searchTerm) => {
```

Comment:
Let's make another component that accept searchTerm as prop. Just to make it more readable